app/models/shipit/stack.rb

```diff
@@ -528,8 +528,25 @@ def sync_github
       GithubSyncJob.perform_later(stack_id: id)
     end
 
+    alias links_orig links
+    def links
+      persisted_tasks = tasks.select(&:persisted?)
+      return links_orig if persisted_tasks.blank?
+
+      variables = Shipit::TaskCommands.new(persisted_tasks.last).env
+      links_orig.each_with_object({}) do |(name, url), replaced_links|
+        replaced_links[name] = replace_variables(url, variables)
+      end
+    end
+
     private
 
+    def replace_variables(string, variables)
```

Comment:
We already have code for that: https://github.com/Shopify/shipit-engine/blob/4192f63825a6e7cf8f437cb2fa0f626ad60f4cf2/lib/shipit/command.rb#L48-L55

Might just need to be extracted to it's own module.

Reply:
Addressed by https://github.com/Shopify/shipit-engine/pull/1018/commits/671d045de9eb4d64cd2e4dcc9e7bb8a9ac5cdf97